config/names.coffee

```diff
@@ -14,8 +14,8 @@ DOMAIN_ENGINE = process.env.DOMAIN_ENGINE || 'balena.io/engine'
 FORUMS_DOMAIN = process.env.DOMAIN_FORUMS || 'https://forums.balena.io'
 BASE_IMAGES_LIB = process.env.BASE_IMAGES_LIB || 'balenalib'
 BASE_IMAGES_CORE = process.env.BASE_IMAGES_CORE || 'balena'
-CLI_LOWER = process.env.CLI_LOWER || 'balena CLI'
-CLI_UPPER = (CLI_LOWER.charAt(0).toUpperCase() + CLI_LOWER.slice(1)) || 'Balena CLI'
+CLI_LOWER = process.env.CLI_LOWER || 'balenaCLI'
+CLI_UPPER = (CLI_LOWER.charAt(0).toUpperCase() + CLI_LOWER.slice(1)) || 'BalenaCLI'
```

Comment:
A second commit styles "balena CLI" as "balenaCLI" (without space), and drops `'the'` when referring to it. Flowdock: https://www.flowdock.com/app/rulemotion/r-product/threads/xrtkAC3uWDMJ4-4GkJL_Hmn8Jvb

Reply:
I have now removed this commit to make this PR easier to review, focusing on the CLI installation instructions. I will create a separate PR for the renaming.